draft/src/api.rs

```diff
@@ -20,8 +22,36 @@ pub trait Drafter {
 #[derive(Debug)]
 pub struct DraftInfo {
     pub card_list: Vec<Card>,
-    pub num_drafters: u32,
-    pub num_packs: u32,
-    pub cards_per_pack: u32,
+    pub num_drafters: usize,
+    pub num_phases: usize,
+    pub cards_per_pack: usize,
```

Comment:
This was Sean's advice - usize is most convenient in general because it's the only one you're allowed to use for indexing into Vecs/Arrays. Also, whenever you do any math with mixed numeric types, you have to cast. So unless there's a reason to require the exact architecture-independent size of the numeric variable, he recommended using usize for everything.

Casting is required since I use a -1 in the code later, but with these as u32 more casting is required elsewhere.

Reply:
Ahh, that makes sense for indexing. Thanks for the info!